### DISCUSSION TITLE
v5.1

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#564] [New releases]
Long time no see! 😅 Hope you're ready for a big release, with no fewer than **four new translations**!  🎉 

📐 To upgrade **from v5.0**, please follow the [instructions in the README](https://github.com/anki-geo/ultimate-geography/blob/master/README.md#upgrading). If you're still on **v4.1 or earlier**, please follow the special [upgrade instructions](https://github.com/anki-geo/ultimate-geography/wiki/Upgrade-instructions) on the wiki instead.

⚠️ For [inconsequential reasons](https://github.com/anki-geo/ultimate-geography/discussions/563#discussioncomment-3409995), CrowdAnki will bring up the "Change Note Type" dialog, so be careful to select the _Ultimate Geography_ note type in the drop-down menu. Apart from that, everything should go smoothly. If you run into issues, please let us know in the v5.1 [discussion thread](https://github.com/anki-geo/ultimate-geography/discussions/).

### New translations

- 🎉 **[PT] Portuguese** #489 #518 - by @Kitsune912 
- 🎉 **[ZH] Chinese** (simplified) #493 #499 #519 #524 - by @ruin1990
- 🎉 **[PL] Polish** #538 #552 #555 #559 - by @Astilimos 
- 🎉 **[IT] Italian** #554 #558 - by @giovanni-cutri

### Country names, capitals and supplementary info

- 👉 Change capital of Kiribati from Tarawa to **South Tarawa** #166 #512 (see also #416 and [_Capital field_ guidelines](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#capital-field))
- 👉 [DE] Change Weißrussland to **Belarus** and mention alternative name in country info #485 #486
- 👉 [ES] Change spelling of capital of Kazakhstan from Nur-sultán to **Nursultán** #508 
- 👉 [NL] Help distinguish between **Sint Maarten** and **Sint-Maarten**, the Dutch and French territories on the island of Saint-Martin #464 #467
- 👉 [NL] Mention **Belarus** as an alternative name for Wit-Rusland #527 
- ⛑️ [NL] Fix spelling of **Bosnië en Herzegovina** and multiple other minor typos #496 #500 #501 #505 #497 #526 #528 #529 #530 #533 
- ⛑️ [RU] Fix multiple country and capital names and fix/improve various country/capital info #474 #515 #525 
- [DE] Remove `DARS` acronym from country name of Sahrawi Arab Democratic Republic #547 
- [DE] Fix gender typo in country name of Canary Islands #547 
- [DE] Fix small typos and make minor improvements #483 #484 #514 
- [ES] Fix minor gender typos #487
- [FR] Fix exact month when Kazakhstan renamed its capital #557 

### Maps and flags

- 👉 Replace flag of Afghanistan (tricolor flag outlawed by the Taliban) #531 
- ✨ Add contextual zoom box to map of Gibraltar #461 #507 
- Remove longitude and latitude lines from Ukraine map #471 #482
- Update flag of Mauritania (minor colour and geometrical changes) #531 
- Update flag of Eswatini (colour of feathers) #506 

### Other changes

- Remove tag `UG::European_Union` from note of the United Kingdom #536
- Translate deck description to Spanish #510 #511 
- Document choices made regarding Brazilian and European dialects for Portuguese translation in [CONTRIBUTING guide](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#portuguese) #490 
- Clarify [major version upgrade instructions](https://github.com/anki-geo/ultimate-geography#major-version) in README to mention that "clean import" is usually not the only way #479 
- Recommend using a text editor instead of spreadsheet in CONTRIBUTING guide to avoid mangling GUIDs #539 
- Upgrade to BrainBrew 0.3.8 (from 0.3.2), remove legacy recipes and extract note models into their own YAML files #502 #537 #548 #553 
- Fix typos in CONTRIBUTING guide and README #544
- Optimise images with [Imgbot](https://github.com/marketplace/imgbot) by 15% all up (67 kB) #546 

<hr /><em>This discussion was created from the release <a href='https://github.com/anki-geo/ultimate-geography/releases/tag/v5.1'>v5.1</a>.</em>

**Comment by jeprecated** [comment]
Great effort mate 🎉 👏

**Comment by nicolopz** [comment]
How could I use two decks, like English and Spanish? I have been using the English extended version for a while and wanted to use the Spanish version too. However, when I download the Spanish version and import it using CrowdAnki, my English deck gets translated to Spanish.

**Reply by aplaice** [reply]
By "my English deck gets translated to Spanish" do you mean that the deck description is translated to Spanish or that your existing cards/notes are translated to Spanish?

<hr/>

What I'd expect to happen when you import the Spanish deck after having imported the English deck:

1. You have one single "Ultimate Geography" deck in Anki.
2. This deck's description is in Spanish.
3. That one single deck contains all the cards/notes twice, once in English once in Spanish.

If this "expected" behaviour is what's happening, but you don't like please complain! If this is _not_ what's happening please complain extra hard!

<hr/>

If you want to avoid this (have two separate decks, one with English, one with Spanish) you need to modify the first `crowdanki_uuid` in the Spanish deck (to anything different) and ideally change the name (on line 605). (The first change is necessary, the second is useful for aesthetic purposes (otherwise you'll end up with a deck called something like "Ultimate Geography_2").) For instance you could make the following changes:

```diff
@@ -1,7 +1,7 @@
 {
     "__type__": "Deck",
     "children": [],
-    "crowdanki_uuid": "43c5ba66-9a65-11e8-90c9-a0481cc15658",
+    "crowdanki_uuid": "43c5ba66-9a65-11e8-90c9-a0481cc15659",
     "dyn": 0,
     "extendNew": 10,
     "extendRev": 50,
@@ -602,7 +602,7 @@
         "ug-map-zanzibar.png",
         "ug-map-zimbabwe.png"
     ],
-    "name": "Ultimate Geography",
+    "name": "Ultimate Geography (ES)",
     "note_models": [
         {
             "__type__": "NoteModel",
```

You can carry out the import over the previous import, if you haven't reset from a backup (the Spanish cards will be moved to the new deck (though the deck description of the "original" English deck will remain in Spanish (to remedy that you can either change it manually or re-import the English deck))) or as a fresh import, if you had reset from a backup.

(In either case importing to the collection with your existing English deck.)

**Reply by nicolopz** [reply]
Yes, the deck's description turned into Spanish and now I have one deck with both cards in Spanish and in English.

I made the changes as you suggested, and now I have two decks. Thank you so much! And thank you for sharing this deck with the community.

**Comment by DeadliftAndBeer** [comment]
I've used to the version I downloaded from Ankiweb, which I think was 5.1 and been using for a while. However I decided to get the Swedish Deck since I'm mainly interested in learning The Swedish names of countries and capitals. I followed the instructions and ended up with two decks. However the cards where not moved to the new deck as per the instructions. The new deck with the Swedish card had no review history. Am I not supposed to upgrade to Swedish and keep the progress I've done in the original version?

**Reply by aplaice** [reply]
The intention is to allow people to have both decks — English and Swedish — in parallel.  (Or in general, to allow them to have any number of translated decks, in parallel.) Hence, yes, the new deck with the Swedish names should have no review history — the Swedish-language notes/cards should be separate from their English versions.

(This is indeed different from the behaviour of upgrading the "standard" deck to the "extended" one, where the 

<hr/>

If you really want the Swedish versions to replace their English counterpart, then this is doable with some "magic". The attached deck should do the trick. i.e.:

[Ultimate_Geography_SV.zip](https://github.com/anki-geo/ultimate-geography/files/12233091/Ultimate_Geography_SV.zip)

1. Create a backup (just in case).
2. Delete the new (recently created) deck.
3. Import the new deck.

(The attached SV deck differs from the original 5.1 by the GUIDs of the notes — they're the same as those of the English deck.)

**Reply by DeadliftAndBeer** [reply]
Thanks, worked wonderfully
